config/locales/en/admin.yml

```
@@ -1638,6 +1638,7 @@ en:
       index:
         create: Create tenant
       new:
+        admin_information: 'When you create a tenant, your current user <b>"%{username}"</b> with the email  <b>"%{email}"</b> will be copied into the database of the new tenant and will be automatically granted administration permissions.'
```

I'm not so fond of `<b>` tags (although it's true that we use it sometimes, for whatever reasons :relieved:). We usually use `<strong>` when we'd like to note that a certain element isn't important. If we only want to make the text bold, we use a `<span>` tag and style it with CSS.

By the way, there are two spaces after "the email".

```suggestion
        admin_information: 'When you create a tenant, your current user <strong>"%{username}"</strong> with the email <strong>"%{email}"</strong> will be copied into the database of the new tenant and will be automatically granted administration permissions.'
```